.github/actions/send-tweet/action.yml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'Send Tweet Action'
+description: 'Announce a Firebase release via Twitter.'
```

Comment:
Nit: this action can be used by anyone to post a tweet using github actions, right? Do we want to generalize the description here?

Reply:
Good idea. Done.